phoenix-core/src/main/java/org/apache/phoenix/monitoring/MetricType.java

```diff
@@ -25,6 +25,7 @@
     QUERY_TIME("Query times"),
     NUM_PARALLEL_SCANS("Number of scans that were executed in parallel"),
     SCAN_BYTES("Number of bytes read by scans"),
+    SNAPSHOT_SCAN_BYTES("Number of bytes read by snapshot scans"),
```

Comment:
Do we need a different metric here? Or would it make more sense to use SCAN_BYTES for the snapshot case too?